pyartcd/pyartcd/jenkins.py

```diff
@@ -56,10 +56,15 @@ async def trigger_rhcos(build_version: str, new_build: bool):
     )
 
 
-async def trigger_build_sync(build_version: str):
+async def trigger_build_sync(build_version: str, doozer_data_gitref: str = '',
+                             triggered_from_gen_assembly: bool = False):
     await trigger_jenkins_job(
         job_path='job/triggered-builds/job/build-sync',
-        params={'BUILD_VERSION': build_version}
+        params={
+                    'BUILD_VERSION': build_version,
+                    'DOOZER_DATA_GITREF': doozer_data_gitref,
+                    'TRIGGERED_FROM_GEN_ASSEMBLY': triggered_from_gen_assembly
```

Comment:
This func calls this job https://github.com/openshift-eng/aos-cd-jobs/blob/master/triggered-jobs/build-sync/Jenkinsfile so all modifications would need to go here as well

Reply:
Yes, I did make that changes as well, its part of this [PR](https://github.com/openshift-eng/aos-cd-jobs/blob/77a1148eb0b93829baaaee5facb10ed9fb4174b6/jobs/build/build-sync/Jenkinsfile) as well. Not sure if I need to add something different?

Reply:
Ohh I see, was looking at the build sync job, got it will make the change

Reply:
Updated and repushed